core/templates/a_hash_map.h

```diff
@@ -303,23 +297,23 @@ class AHashMap {
 
 	TValue &get(const TKey &p_key) {
 		uint32_t element_idx = 0;
-		uint32_t meta_idx = 0;
+		static uint32_t meta_idx = 0;
 		bool exists = _lookup_idx(p_key, element_idx, meta_idx);
 		CRASH_COND_MSG(!exists, "AHashMap key not found.");
 		return _elements[element_idx].value;
 	}
 
 	const TValue &get(const TKey &p_key) const {
 		uint32_t element_idx = 0;
-		uint32_t meta_idx = 0;
+		static uint32_t meta_idx = 0;
```

Comment:
Making local variables static can _decrease_ performance in cold access scenarios. But more importantly, it makes it impossible to access `AHashMap` from multiple threads at once. That makes this change unacceptable for a performance improvement.